### ISSUE TITLE
[9.3.2] The SLA/SLT fields hidden in the ticket templates are shown on the ticket page

### ISSUE THREAD

**Issue by pedroluisvsousa** [glpi-project/glpi#4842]
The SLA/SLT fields hidden in the ticket templates are shown on the ticket page.

* Steps to reproduce (which actions have you made) :

1. Create a Business Rule to set SLA/SLT;
2. In the a ticket template set SLA/SLT tab as hidden;
3. Open some ticket in the assistance menu.

* Actual result :
It shows the SLA/SLT fields

* Expected result :
No show the SLA/SLT fields

* Screenshot of the problem (if pertinent) :
![image10](https://user-images.githubusercontent.com/26510405/47650106-f98cdd00-db5d-11e8-9a13-2c263b8397dc.png)
![image11](https://user-images.githubusercontent.com/26510405/47650112-fbef3700-db5d-11e8-9855-480009b14fd0.png)

* Your GLPI setup (you can find it in Setup > General menu, System tab) :
```

[code]   GLPI 9.3.2 (/nti932 => /var/www/html/nti932) Installation mode: TARBALL
--

Operating system: Linux localhost.localdomain 3.10.0-862.11.6.el7.x86_64 #1 SMP Tue Aug 14 21:49:04 UTC 2018 x86_64 PHP 7.2.11 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apcu, bz2, calendar, 	ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, imap, intl, json, ldap, libxml, mbstring, mysqli, 	mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, posix, session, shmop, sockets, sqlite3, standard, sysvmsg, sysvsem, sysvshm, 	tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib) Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files" 	upload_max_filesize="2M"  Software: Apache/2.4.6 (CentOS) PHP/7.2.11 () 	Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:63.0) Gecko/20100101 Firefox/63.0 Server Software: MySQL Community Server (GPL) 	Server Version: 5.7.23 	Server SQL Mode:  	Parameters: glpiuser@localhost/nti93 	Host info: Localhost via UNIX socket 	 mysqli extension is installed ctype extension is installed fileinfo extension is installed json extension is installed mbstring extension is installed zlib extension is installed curl extension is installed gd extension is installed simplexml extension is installed xml extension is installed ldap extension is installed imap extension is installed Zend OPcache extension is installed APCu extension is installed xmlrpc extension is installed CAS extension is installed Database version seems correct (5.7.23) - Perfect! /var/www/html/nti932/files/_log : OK /var/www/html/nti932/config : OK /var/www/html/nti932/files : OK /var/www/html/nti932/files/_dumps : OK /var/www/html/nti932/files/_sessions : OK /var/www/html/nti932/files/_cron : OK /var/www/html/nti932/files/_graphs : OK /var/www/html/nti932/files/_lock : OK /var/www/html/nti932/files/_plugins : OK /var/www/html/nti932/files/_tmp : OK /var/www/html/nti932/files/_cache : OK /var/www/html/nti932/files/_rss : OK /var/www/html/nti932/files/_uploads : OK /var/www/html/nti932/files/_pictures : OK Web access to the files directory should not be allowed Check the .htaccess file and the web server configuration. SELinux mode is Disabled

htmLawed version 1.2.4 in (/var/www/html/nti932/lib/htmlawed) SimplePie version 1.5.2 in (/var/www/html/nti932/vendor/simplepie/simplepie/library) TCPDF version 6.2.25 in (/var/www/html/nti932/vendor/tecnickcom/tcpdf) michelf/php-markdown in (/var/www/html/nti932/vendor/michelf/php-markdown/Michelf) true/punycode in (/var/www/html/nti932/vendor/true/punycode/src) iacaml/autolink in (/var/www/html/nti932/vendor/iamcal/lib_autolink) sabre/vobject in (/var/www/html/nti932/vendor/sabre/vobject/lib) phpCas version 1.3.5 in (/usr/share/pear)

Server: 'srvappv5.jfce.jus.br', Port: '389', BaseDN: 'ou=People,dc=jfce,dc=jus,dc=br', Connection filter: none, RootDN: 		'uid=samba,ou=People,dc=jfce,dc=jus,dc=br', Use TLS: none

Not active

Way of sending emails: SMTP+SSL (glpiteste01@gmail.com@smtp.gmail.com)

Name: 'glpiteste01@gmail.com' Active: Yes 	Server: '{imap.gmail.com:993/imap/ssl}' Login: 'glpiteste01' Password: Yes

behaviors            Name: Comportamental                 Version: 2.1.1      State: To be cleaned 	escalade             Name: Escalonamento                  Version: 2.3.2      State: To be cleaned 	formcreator          Name: Formulários                    Version: 2.6.4      State: To be cleaned 	mod                  Name: GLPI Modifications             Version: 1.2.3      State: To be cleaned 	satisfaction         Name: More satisfaction              Version: 1.3.1      State: To be cleaned 	moreticket           Name: More ticket                    Version: 1.4.3      State: To be cleaned 	dashboard            Name: Painel                         Version: 0.9.4      State: To be cleaned
[/code]

```

**Comment by Badoche1337** [comment]
Please test fix proposed in #4849

**Comment by pedroluisvsousa** [comment]
Tested. That's right. Thank you.

**Comment by Armin83** [comment]
FYI this bug is in 9.3.1 too.

**Comment by trasher** [comment]
Fixed in current 9.3/bf and master branches
